prow/report/report.go

```diff
@@ -107,22 +111,35 @@ func Report(ghc GithubClient, reportTemplate *template.Template, pj v1.ProwJob)
 	if ghc == nil {
 		return fmt.Errorf("trying to report pj %s, but found empty github client", pj.ObjectMeta.Name)
 	}
+
 	if !pj.Spec.Report {
 		return nil
 	}
+
 	refs := pj.Spec.Refs
-	if len(refs.Pulls) != 1 {
-		return fmt.Errorf("prowjob %s has %d pulls, not 1", pj.ObjectMeta.Name, len(refs.Pulls))
+	if len(refs.Pulls) > 1 {
+		return fmt.Errorf("prowjob %s has %d pulls, thus will skip report", pj.ObjectMeta.Name, len(refs.Pulls))
```

Comment:
Isn't this expected in the case of a batch, rather than an error?

Reply:
we were not reporting batch jobs before (the check was != 1), do we want to change that behavior?

Reply:
I agree with not reporting, I'm just suggesting we don't return an error for the "do nothing because it's a batch"...

Reply:
done